### ISSUE TITLE
Odd annotation in Modelica.Electrical.MultiPhase.Sources.SignalVoltage

### ISSUE THREAD

**Issue by modelica-trac-importer** [modelica/Modelica#352]
**Reported by anonymous on 13 Apr 2010 14:41 UTC**

```
model SignalVoltage "Multiphase signal voltage source"
    ...
     annotation (
        Icon(graphics={
            ...
    Text(
              extent={{-100,100},{-20,60}},
              lineColor={0,0,0}),
    ...
end SignalVoltage;
```

Notice that there is a Text annotation containing no text string.  Probably either a string was intended to be added or that Text annotation was to have been removed.

----

Migrated-From: https://trac.modelica.org/Modelica/ticket/352

**Comment by modelica-trac-importer** [comment]
**Comment by AHaumer on 27 Apr 2010 19:39 UTC**
resolved by f316089a161164101221191de64ee666b2ac7306

**Comment by modelica-trac-importer** [comment]
**Modified by dietmarw on 28 Apr 2010 06:37 UTC**
